e2e/testcases/stress_test.go

```diff
@@ -253,8 +255,7 @@ func TestStressLargeRequest(t *testing.T) {
 		nomostest.WithRootSha1Func(nomostest.RemoteRootRepoSha1Fn),
 		nomostest.WithSyncDirectoryMap(map[types.NamespacedName]string{
 			nomostest.DefaultRootRepoNamespacedName: "configs",
-		}),
-		nomostest.WithTimeout(30*time.Minute))
```

Comment:
Why remove this? Doesn't this lengthen the test timeout? Don't we still need that?

Reply:
I removed it since this == the default wait timeout for the stress tests now